.github/workflows/release.yml

```diff
@@ -51,6 +51,36 @@ jobs:
         with:
           name: Benchmark-Binaries
           path: benchmark-binaries
+      - name: Upload Release Asset AArch64
+        if: github.event_name == 'release'
```

Comment:
Isn't the fact that this is in `release.yml` enough to only be triggered on releases?

Reply:
This workflow can also be triggered by push, so this check prevent's the step from running:
https://github.com/larq/compute-engine/blob/1756d2969d17ccd68a711a742e331c18d3eea3b0/.github/workflows/release.yml#L8-L10